exercises/secret-handshake/introduction.md

```diff
@@ -0,0 +1,11 @@
+# Introduction
+
+There's a joke that goes:
+
+_There are 10 types of people in the world._
+_There are those who understand binary, and those who don't._
+
+You belong to the group that understands binary, and you and your binary-savvy friends want to create a secret handshake.
+You don't want anyone who isn't in the know to be able to crack the code.
+
+So you come up with a sequence of actions, where each action corresponds to a different binary digit.
```

Comment:
Can we add a exercism/note that says "No idea what binary is or how to use it? Check this tutorial..."

Reply:
I searched for something by Vaidehi, and found this: https://medium.com/basecs/bits-bytes-building-with-binary-13cb4289aafa

Look good?